mdata/cache/ccache.go

```diff
@@ -125,20 +126,20 @@ func (c *CCache) CacheIfHot(metric string, prev uint32, itergen chunk.IterGen) {
 	met.Add(prev, itergen)
 }
 
-func (c *CCache) Add(metric, rawMetric string, prev uint32, itergen chunk.IterGen) {
+func (c *CCache) Add(metric, rawMetric schema.AMKey, prev uint32, itergen chunk.IterGen) {
```

Comment:
Does this still require both params `metric` and `rawMetric`? I think just one `schema.AMKey` contains all the information that's needed, because the raw key can be extracted from it.

Reply:
you're right